src/draw.c

```diff
@@ -391,7 +391,11 @@ static int layout_get_height(colored_layout *cl)
         return MAX(h, h_icon);
 }
 
-static cairo_surface_t *render_background(cairo_surface_t *srf, colored_layout *cl, colored_layout *cl_next, int y, int width, int height, bool first, bool last, int *ret_width)
+static cairo_surface_t *render_background(cairo_surface_t *srf,
+                                          colored_layout *cl, colored_layout *cl_next,
```

Comment:
I have to out myself: I'd rather take the overlong line, than a two-dimensional parameter-list. In readability this is actually a disservice.

IMHO think the gnome guys do it best [(example)](https://git.gnome.org/browse/libnotify/tree/libnotify/notification.c?h=0.7.7#n115).

Reply:
I think it takes a bit too much space now but I agree it is more readable that way.